extensions/wikia/AdEngine/js/lookup/prebid.js

```diff
@@ -15,7 +16,7 @@ define('ext.wikia.adEngine.lookup.prebid', [
 		bidderKey = 'hb_bidder',
 		bidKey = 'hb_pb',
 		sizeKey = 'hb_size',
-		url = '//acdn.adnxs.com/prebid/prebid.js';
+		url;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the declaration of the unused 'url' variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3AUnusedVariable)